buffer.go

```diff
@@ -42,7 +45,7 @@ func (s *buffer) PutAggregated(name string, value float64, tags Tags, timestamp
 		s.aggBuf[aggregation] = make(map[AggregationFrequency][]string)
 	}
 
-	s.aggBuf[aggregation][frequency] = append(s.aggBuf[aggregation][frequency], MetricToString(name, value, tags, timestamp, Aggregations{}, 0))
+	s.aggBuf[aggregation][frequency] = append(s.aggBuf[aggregation][frequency], MetricToString(name, value, "", tags, timestamp, Aggregations{}, 0))
```

Comment:
Any reason why we are not adding the put options functionality on the `PutAggregated` method as well? (And on the corresponding client method as well of course)